test/unit/coffee/S3PersistorManagerTests.coffee

```diff
@@ -50,121 +59,95 @@ describe "S3PersistorManagerTests", ->
 		@S3PersistorManager = SandboxedModule.require modulePath, requires: @requires
 
 	describe "getFileStream", ->
-		beforeEach ->
-			@opts = {}
+		describe "success", ->
+			beforeEach () ->
+				@expectedStream = { expectedStream: true }
+				@s3Request.send.callsFake () =>
+					@s3EventHandlers.httpHeaders(200, {}, @s3Response, "OK")
+				@s3Response.httpResponse.getUnbufferedStream.returns(@expectedStream)
+
+			it "returns a stream", (done) ->
+				@S3PersistorManager.getFileStream @bucketName, @key, @opts, (err, stream) =>
+					if err?
+						return done(err)
+					expect(@awsS3.lastCall.args).to.deep.equal([{
```

Comment:
This is a test on the credentials, while a test failure would hint that something other that a stream was returned.